pkgs/by-name/ta/tact/package.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  pkgs,
+  stdenv,
+  fetchYarnDeps,
+  yarnConfigHook,
+  yarnBuildHook,
+  yarnInstallHook,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tact";
+  version = "1.6.7";
+  src = fetchFromGitHub {
+    owner = "tact-lang";
+    repo = "tact";
+    tag = "v${version}";
```

using finalAttrs is preferred when supported 
```suggestion
stdenv.mkDerivation (finalAttrs: {
  pname = "tact";
  version = "1.6.7";
  
  src = fetchFromGitHub {
    owner = "tact-lang";
    repo = "tact";
    tag = "v${finalAttrs.version}";
```

---

Fixed, thank you